app/src/main/res/values/strings.xml

```diff
@@ -40,4 +40,8 @@
     <string name="menu_cfilters">Use CFilters (experimental)</string>
     <string name="other">Other</string>
     <string name="menu_other_description">Enter an url.</string>
+    <string name="dcrdata_warning_alert_title">Warning!</string>
+    <string name="dcrdata_warning_alert_details">Ticket transaction IDs and/or addresses will be sent over the internet to dcrdata in order to fetch balances and statuses. A bad actor may be able to spy on you and link together parts of a transaction history. This is a privacy concern. Only continue if you accept the risks.</string>
```

Comment:
- `internet` -> `Internet`
- "parts of a transaction history" -> "parts of your transaction history"
- if you spell out the specific dcrdata domain it would help to better assess the risk based on the perceived reputation of that domain's owners
- does it only check for _ticket_ tx/addrs or regulars too?

Reply:
Any addrs for balance.

Reply:
![image](https://user-images.githubusercontent.com/34998433/114800206-39021f00-9dd4-11eb-88ce-f4a0d25fc0cb.png)

Reply:
I only planned on showing this once for the lifetime of the app, so not every time. Sounds ok?

Reply:
I didn't think that a full URL will look awkward. Maybe drop URL schema and path `https://explorer.dcrdata.org/api/` -> `dcrdata.org`. My motivation for showing the domain is to help with the trust decision, and `dcrdata.org` should be enough.

Also `of your transaction history` reads slightly smoother and rings the alarm bell a bit better.

Reply:
How now?

Reply:
Nice!